pkg/controller/core.oam.dev/v1alpha2/application/generator.go

```diff
@@ -301,12 +332,24 @@ func getComponentResources(ctx context.Context, manifest *types.ComponentManifes
 	return workload, traits, nil
 }
 
-func generateStepID(stepName string, wfStatus *common.WorkflowStatus) string {
+func generateStepID(stepName string, wfStatus *common.WorkflowStatus, parentStepName string) string {
```

Comment:
We can make this function to `generateStepID` and `generateSubStepID` to make this clear.

Reply:
Ok